cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/internal/BootstrapCoordinator.scala

```diff
@@ -171,10 +172,8 @@ private[akka] final class BootstrapCoordinator(discovery: SimpleServiceDiscovery
       // the next round of discovery will be performed once this one returns
       discoverContactPoints()
 
-    case SimpleServiceDiscovery.Resolved(name, contactPoints) ⇒
-      serviceName = name
-
-      log.info("Located service members with name: [{}]: [{}]", serviceName, contactPoints.mkString(", "))
+    case SimpleServiceDiscovery.Resolved(_, contactPoints) ⇒
+      log.info("Located service members with name: [{}]: [{}]", lookup, contactPoints.mkString(", "))
```

Comment:
Perhaps make the message something like "Looked up service members based on [{}]: [{}]"?